.github/workflows/build-docker.yml

```diff
@@ -1,22 +1,29 @@
-name: Build and push off-chain docker image
+name: Build and deploy off-chain 
 
```

Comment:
Rename secrets in order to be consistent between the repositories:
`DOCKER_USERNAME` -> `DOCKERHUB_USERNAME`
`DOCKER_PASSWORD` -> `DOCKERHUB_TOKEN`; to use access token instead of password.